docs/assignment1.md

```diff
@@ -1,62 +1,58 @@
+Other features we will implement if time permits are multiplayer and persistence. For multiplayer, we will take the game one step further by implementing a Java chat room using ServerSocket. Because text adventure games are basically semi-automated chat rooms between a player and the prewritten consequences of various actions and objects in the game, we believe we can add other users to this “chat room” to create multiplayer functionality which will depend on the overall objective of the game. The persistence feature will be significantly easier. In order to accomplish this, we will include a specification of where the player currently is and which objects they are holding in the JSON file that was initially inputted into the generic game. When the player closes the terminal, this information will save to this file, so if the same file is again inputted when the terminal is later opened, they can pick up where they left off.
 
-Be creative here!
-
-Don’t forget to use links to external URLs (e.g., the direct link to the Fantasy soccer league you are getting inspiration from, the link to the original video game you are getting inspiration from, etc.), if applicable. 
-
-Maximum number of words for this section: 1000
 
 ## Features
-Author(s): `name of the team member(s) responsible for this section`
-
-<When defining both functional features and quality requirements, remember that you will need to come back to them in Assignments 2 and 3 and explicitly state how specific parts of models/implementation satisfy them.>
+Author(s): Beth, Sarah, Ben, Sam, Gemma
 
 ### Functional features
 
-As a preamble to the table, you can discuss the main line of reasoning you applied for defining the functional features provided in the table.
+In order to make the game as generic as possible, we want to include Functional Features that will be used in the game such as Commands, Movements, etc. that the player will be able to use during the game. 
 
 | ID  | Short name  | Description  |
 |---|---|---|
-| F1  | Tags | Code snippets can be tagged via freely-defined labels called tags  |
-| F2  | Commands  | The player can control the main character by issuing command-line commands following this syntax: `command-name [target-objects]*`. The available `command-names` are the following: <br/> - move: ... <br/> - use: ... <br/> - inspect: ... <br/> |
-| F3  | Movements  | The main character shall move freely in the environment according  |
-| F4  | ... | ... |
+| F1  | Command | Player can control the main character using command line commands with the following syntax: ‘[command] [object]’. The commands we will implement are: move, open, break, inspect, help   |
+| F2  | Movements  | Player can move through the environment using command: ‘move to [location]’  |
+| F3  | Help Messages  | After a certain amount of time, user will be given hints of what actions to perform or movements to make  |
```

Comment:
Could have been more precise. Will the hints guide the player specifically to the final location that wins the game or will they have more of a  "trivia" use? i.e the player stands idle in front of an object and gets a help message providing info on how to interact with said object.